src/stream/stream/mod.rs

```diff
@@ -313,6 +315,41 @@ extension_trait! {
             TakeWhile::new(self, predicate)
         }
 
+        #[doc = r#"
+            Limit the amount of items yielded per timeslice in a stream.
+
```

Comment:
It'd be nice to add some more information that this stream does not drop items, and is intended for backpressure only.

```suggestion

This stream does not drop any items, but will only limit the rate at which items pass through.
```